presto-array/src/main/java/com/facebook/presto/array/Arrays.java

```diff
@@ -24,6 +24,8 @@
 
 public class Arrays
 {
+    private static final double graceFactorFordMaxCapacity = 1.2f;
```

Comment:
- all caps with underscores
- consider making this configurable

Reply:
@mbasmanova I will send a separate PR to make it configurable.